src/js/lib/monster.ui.js

```diff
@@ -984,10 +984,12 @@ define(function(require) {
 				inputEndDate = container.find('#endDate'),
 				now = new Date(),
 				today = new Date(now.getFullYear(), now.getMonth(), now.getDate(), 0, 0, 0, 0),
-				startDate = new Date(now.getFullYear(), now.getMonth(), now.getDate(), 0, 0, 0, 0),
+				startDate = options.startDate || new Date(now.getFullYear(), now.getMonth(), now.getDate(), 0, 0, 0, 0),
```

Comment:
You can use `lodash#get` for a more robust path check here:
```suggestion
				startDate = _.get(
					options,
					'startDate',
					new Date(now.getFullYear(), now.getMonth(), now.getDate(), 0, 0, 0, 0)
				);
```